aten/src/ATen/core/CachingHostAllocator.h

```diff
@@ -343,7 +345,7 @@ struct CachingHostAllocatorImpl {
     TORCH_CHECK_NOT_IMPLEMENTED(false, "Not implemented for copy_data");
   }
 
-  HostStats getStats() {
+  HostStats get_stats() {
```

Comment:
I'm not sure about renaming all these functions. CamelCase is pretty standard in our C++ codebase so I think it's best to avoid the churn of having to change call sites !

Reply:
@guangyey  It's not necessary to rename all these functions, right?